x/poolmanager/create_pool.go

```diff
@@ -85,6 +85,8 @@ func (k Keeper) CreatePool(ctx sdk.Context, msg types.CreatePoolMsg) (uint64, er
 		return 0, err
 	}
 
+	k.poolCreationListeners.AfterPoolCreated(ctx, sender, pool.GetId())
```

Comment:
From initial scan, I don't think this is being actually implemented any where, can you cross check if this is true? If so , can we make an issue to remove this?